scripts/gcloud.sh

```diff
@@ -24,12 +24,14 @@ declare gcloud_region="--region=europe-west6"
 declare gcloud_disk_name="hoprd-data-disk"
 
 GCLOUD_MACHINE="--machine-type=e2-medium"
-GCLOUD_META="--metadata=google-logging-enabled=true,google-monitoring-enabled=true,enable-oslogin=true --maintenance-policy=MIGRATE"
+GCLOUD_META="--metadata=google-logging-enabled=true,google-monitoring-enabled=true,enable-oslogin=true --maintenance-policy=TERMINATE"
 GCLOUD_TAGS="--tags=hopr-node,web-client,rest-client,portainer,healthcheck"
 GCLOUD_BOOTDISK="--boot-disk-size=20GB --boot-disk-type=pd-standard"
 GCLOUD_IMAGE="--image-family=cos-stable --image-project=cos-cloud"
+GCLOUD_SCOPES="--scopes=https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append,https://www.googleapis.com/auth/compute"
```

Comment:
Why is this needed now?

Reply:
When we pass `--preemptible` to machines, they get disposed of by Google's scheduler. In short, this value can't be `MIGRATE`. An alternative could be to _only_ change this if we pass the `--preemptible` flag

![image](https://user-images.githubusercontent.com/1128312/140491150-21ac2f9d-23d0-469d-9126-7de48c648f44.png)

Reply:
I mean the scopes.